bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/RowRangeAdapter.java

```diff
@@ -67,7 +69,7 @@
         break;
       default:
         throw new IllegalArgumentException("Unexpected start key case: " +
-            btRange.getStartKeyCase());
+                btRange.getStartKeyCase());
```

Comment:
Please fix your spacing to be similar to the current style.  There should be an eclipse style xml file at the root of the project.